measure.rb

```diff
@@ -2258,10 +2258,12 @@ def self.add_cooling_system(runner, model, building)
 
         if num_speeds == "1-Speed"
 
-          eers = [0.82 * seer + 0.64]
           shrs = [0.73]
+          capacity_ratios = [1.0]
+          fan_speed_ratios = [1.0]
```

Comment:
I'd rather go the opposite approach. For all ACs/ASHPs, can you make these assignments inside the apply_foo methods and remove as arguments?